app/templates/guidelines.hbs

```diff
@@ -34,7 +35,7 @@
     </ol>
   </section>
 
-  <section class="margin-top-medium">
+  <section class="lg:col-5">
     <h2>Diversity: Everyone is Welcome</h2>
```

Comment:
This h2 needs an ID so that the containing section can use the value for aria-labelledby.